README.md

```diff
@@ -17,91 +16,50 @@ at ordering arbitrary messages (transactions). It has been designed to operate
 continuously under conditions where there is no bound on message-delivery delay
 and under the assumption that there is a significant probability of malicious
 behavior, making it an excellent fit for blockchain-related applications.
-For more information, check [the paper][paper-link].
+For more information, check [the white paper][paper-link].
 
 This repository contains a Rust implementation of AlephBFT that offers a convenient
 API enabling seamless application to various problems. The prime application of
 the repository is the consensus engine (sometimes called the "finality gadget")
 of the [Aleph Zero blockchain][aleph-node-link].
 
-### Detailed documentation
+The code is split into several rust packages, each having its own directory -
+see the `Cargo.toml` file, which defines the layout of the whole workspace.
+The main package, `aleph-bft`, is located in the `consensus` directory.
+Additionally, every other package has a short README describing its role
+in the AlephBFT toolset.
 
-If the crate's [documentation][docs-link] seems to be not comprehensive enough,
-please refer to the [detailed version][reference-link].
+### Documentation
```

Comment:
I would add a brief instruction how to build docs locally, i.e. `install cargo + cargo install mdbook + cd docs + mdbook serve --open`